project-generated-code-backend/Backend/Dto/FeedbackDTO.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Backend.Model.Util;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace health_clinic_class_diagram.Backend.Dto
+{
+    public class FeedbackDTO
+    {
+        public bool CorrectText()
+        {
+            String[] words = Text.Split('\n');
+            return words != null && Text != null && Text.Length > 2;
+        }
+
+        public bool IsNotapproved()
+        {
+            return Approved != null;
+        }
```

Comment:
This method is really confusing the name suggests that you are checking if Approve is false, so maybe change to IsApprovalValid